keyboards/crkbd/keymaps/drashna/keymap.c

```diff
@@ -113,12 +113,47 @@ const char *read_logo(void);
 void set_keylog(uint16_t keycode, keyrecord_t *record);
 const char *read_keylog(void);
 const char *read_keylogs(void);
-
+char layer_state_str[24];
 // const char *read_mode_icon(bool swap);
-// const char *read_host_led_state(void);
+const char *read_host_led_state(void);
 // void set_timelog(void);
 // const char *read_timelog(void);
 
+
+const char* read_layer_state(void) {
+  switch (layer_state) {
+    case _QWERTY:
+      switch (default_layer_state) {
+        case _QWERTY:
+          snprintf(layer_state_str, sizeof(layer_state_str), "Layer: QWERTY");
+          break;
+        case _COLEMAK:
+          snprintf(layer_state_str, sizeof(layer_state_str), "Layer: Colemak");
+          break;
+        case _DVORAK:
+          snprintf(layer_state_str, sizeof(layer_state_str), "Layer: Dvorak");
+          break;
+        case _WORKMAN:
```

Comment:
wow! you're one of the few I know who use workman!

Reply:
I don't actually.  I've been meaning to try it, since it's not TOO different from colemak.  

But I also design my boards for others to use, as well.